src/metric/from.rs

```diff
@@ -77,8 +77,13 @@ fn _from_value<'f>(
 // thread_pool_transform_indexing_rejected
 //
 // This is inconvenient to use with exporter_include_labels CLI argument
-const SKIP_PREFIX_APPEND: &[&'static str; 4] =
-    &["name", "ip", "host", reserved::INJECT_CLUSTER_VERSION];
+const SKIP_PREFIX_APPEND: &[&'static str; 5] = &[
```

Comment:
nitpicking, any reason why some consts are a reference to arrays with fixed lengths and some are array slices?

Reply:
not really, removed fixed length :+1: